MalawiMeta.Api/Endpoints/Cities/MapAllCities.cs

```diff
@@ -0,0 +1,31 @@
+﻿using MalawiMeta.Api.Extensions;
+using MalawiMeta.Api.TransferObjects;
+using MalawiMeta.Api.UseCases.Cities;
+using Microsoft.AspNetCore.Mvc;
+
+namespace MalawiMeta.Api.Endpoints.Cities;
+
+public static partial class CityEndpoints
+{
+    private static void MapAllCities(this IEndpointRouteBuilder app)
+    {
+        app.MapGet(
+            "/",
+            [ProducesResponseType(typeof(IEnumerable<CityResponseDto>), StatusCodes.Status200OK)]
```

Comment:
I think the better way is to add these attributes [at the end of the route definition](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis?view=aspnetcore-7.0#iresult-return-values)